pkgs/top-level/python-packages.nix

```diff
@@ -6389,6 +6389,16 @@ in {
     disabled = !isPy3k;
   });
 
+  scipy_1_4 = self.scipy.overridePythonAttrs (oldAttrs: rec {
+    version = "1.4.1";
+    src = oldAttrs.src.override {
+      inherit version;
+      sha256 = "0ndw7zyxd2dj37775mc75zm4fcyiipnqxclc45mkpxy8lvrvpqfy";
+    };
+    doCheck = false;
+    disabled = !isPy3k;
+  });
```

Comment:
Versioned attributes in `python-packages.nix` are not allowed! You have to use `packageOverrides` instead. I will have to remove `nanovna-saver` as well if this is not rectified promptly.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/127910